hackathon_site/dashboard/frontend/src/slices/order/adminOrderSlice.ts

```diff
@@ -205,4 +205,29 @@ export const adminOrderTotalSelector = createSelector(
     (orderItems) => orderItems.reduce((accum) => accum + 1, 0)
```

Comment:
Please change name of `adminOrderTotalSelector` to 'adminOrderTotalWithFiltersSelector', or similar.